tests/host/lldb/__init__.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import annotations
+
+import os
+import subprocess
+import sys
+import time
+from pathlib import Path
+from typing import List
+
+from host import TestHost
+from host import TestResult
+from host import _collection_from_pytest
+from host import _result_from_pytest
+
+
+class LLDBTestHost(TestHost):
+    def __init__(
+        self, pwndbg_root: Path, lldbinit_dir: Path, pytest_root: Path, binaries_root: Path
+    ):
+        assert pwndbg_root.exists()
+        assert pwndbg_root.is_dir()
+
+        assert lldbinit_dir.exists()
+        assert lldbinit_dir.is_dir()
+
+        assert pytest_root.exists()
+        assert pytest_root.is_dir()
+
+        assert binaries_root.exists()
+        assert binaries_root.is_dir()
+
+        self._pwndbg_root = pwndbg_root
+        self._lldbinit_dir = lldbinit_dir
+        self._pytest_root = pytest_root
+        self._binaries_root = binaries_root
+
+    def _launch(
+        self,
+        op: str,
+        test_name: str | None,
+        capture: bool,
+        pdb: bool,
+    ) -> subprocess.CompletedProcess[str]:
+        target = self._pwndbg_root / "tests" / "host" / "lldb" / "launch-guest.py"
```

Comment:
So many `/` operator calls, I bet that `self._pwndbg_root / "tests/host/lldb/launch-guest.py"` is probably more readable?

```suggestion
        target = self._pwndbg_root / "tests/host/lldb/launch-guest.py"
```